packages/hello-gsm-calculate/src/stores/StoreContainer.ts

```diff
@@ -2,13 +2,22 @@ import create from 'zustand';
 
 interface StoreType {
   showScoreResult: boolean;
+  freeSemester: string | null;
+  system: string;
```

Comment:
```suggestion
  system: '자유학년제' | '자유학기제';
```

제가 코드를 확인 해보았을때 위 두 값만 이 store에 할당되는 것 같아 이런식으로 정확한 값들로 union 타입을 지정해서 명확하게 store의 타입을 지정하는 것도 좋아보입니다!

Reply:
<img width="824" alt="스크린샷 2022-09-03 오후 11 26 01" src="https://user-images.githubusercontent.com/80191860/188274765-6c91df3e-b023-4d8d-ac21-a40e89fc05c4.png">
저도 처음에 저렇게 하려고 했었지만 localStorage 에 저장할 때 string 형식만 저장할 수 있어서 저렇게 수정했었습니다..

Reply:
네 알겠습니다!